<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_font">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/Servicios.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QFrame" name="destinos">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
border: 1px solid #c7c7c7;
	background-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_vuelta">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(11, 41, 92);</string>
    </property>
    <property name="text">
     <string>Vuelta: 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_estudiantes">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Estud.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_head">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
background-color: rgb(255, 85, 0);
font: 75 20pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>21140 ROGER-HGO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_estudiantes_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_estudiantes_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;v</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Niño</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_estudiantes_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Ad. May</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_cantidad_boletos_estud">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_cobro_estud">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>21.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_cantidad_boletos_normal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_cobro_normal">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>21.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_cantidad_boletos_ninio">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_cantidad_boletos_admayor">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_cobro_ninio">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>21.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_cobro_admayor">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>21.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_estudiantes_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 16pt &quot;Roboto&quot;;
color: rgb(11, 41, 92);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>TOTAL GENERAL:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_a_liquidar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 16pt &quot;Roboto&quot;;
color: rgb(11, 41, 92);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>21.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_cobro_admayor_digital">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>21.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_cobro_estud_digital">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>21.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_cobro_ninio_digital">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>21.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_cantidad_boletos_ninio_digital">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_cantidad_boletos_estud_digital">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>1 </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_cantidad_boletos_normal_digital">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>1 </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_cobro_normal_digital">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>21.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_cantidad_boletos_admayor_digital">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_vuelta_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(12, 49, 112);</string>
    </property>
    <property name="text">
     <string>$ Efec</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vuelta_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(12, 49, 112);</string>
    </property>
    <property name="text">
     <string>Tipo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vuelta_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(48, 110, 188);</string>
    </property>
    <property name="text">
     <string>Total:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cantidad_total_boletos_efectivo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(48, 110, 188);</string>
    </property>
    <property name="text">
     <string>21</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cantidad_total_boletos_digital">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(48, 110, 188);</string>
    </property>
    <property name="text">
     <string>21</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_cobro_efectivo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(48, 110, 188);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>21.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_cobro_digital">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(48, 110, 188);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>21.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_folio_viaje">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(11, 41, 92);</string>
    </property>
    <property name="text">
     <string>FV: 509112100001</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_operador">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(11, 41, 92);</string>
    </property>
    <property name="text">
     <string>Op: Juan Pérez</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vuelta_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(12, 49, 112);</string>
    </property>
    <property name="text">
     <string>Cant. Efec</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vuelta_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(12, 49, 112);</string>
    </property>
    <property name="text">
     <string>Cant. Digi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vuelta_8">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(12, 49, 112);</string>
    </property>
    <property name="text">
     <string>$ Digi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_version_software">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(11, 41, 92);</string>
    </property>
    <property name="text">
     <string>SW: EL.v2.53</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_user">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>203</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Imagenes/User/user/generico.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_fin">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Imagenes/cerrarVuelta.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_cancel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Imagenes/cancelar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_ultimo_folio_efectivo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(11, 41, 92);</string>
    </property>
    <property name="text">
     <string>Ultimo folio venta efectivo: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ultimo_folio_digital">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(11, 41, 92);</string>
    </property>
    <property name="text">
     <string>Ultimo folio venta digital: 0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
